bus/bus.go

```diff
@@ -159,6 +159,7 @@ type bus struct {
 	logger        *zap.SugaredLogger
 	accounts      *accounts
 	contractLocks *contractLocks
+	sectorsCache  *uploadedSectorsCache
```

Comment:
Maybe `uploadingSectors` is a slightly better name since they are not necessarily uploaded yet.